src/comm_tiled.cpp

```diff
@@ -1874,7 +2016,12 @@ void CommTiled::allocate_swap(int n)
 
   pbc_flag = new int*[n];
   pbc = new int**[n];
-  sendbox = new double**[n];
+   //if(mode == SINGLE){ 
```

Comment:
@Adrian-Diaz what's up with these comments?

Reply:
seems to be commented debug statements or something of the sort I forgot to delete.